frontend/src/components/Summary/ImplementationRow.tsx

```diff
@@ -65,20 +65,20 @@ const ImplementationRow = ({
               <Container className="p-0">
                 <Row className="d-flex flex-column gap-2">
                   <Col>
-                    <span>
-                      <b style={{ fontSize: "1rem" }}>failed</b>:&nbsp;
+                    <b style={{ fontSize: "1rem" }}>failed:</b>&nbsp;
```

Comment:
The manual `b` styling here was always a bit off to me, I think while we're here maybe can we change this to be labels? Maybe with a table. Or else maybe a `<hl>`? Or some other idea? I guess `hl` is most obvious to me.

Reply:
`<h1>` is too big for this purpose, going with `<h6>` instead.